Module1/README.md

```diff
@@ -80,3 +80,49 @@ Use the following to run specific test: e.g.
 ```powershell
 $ poetry.exe run pytest -s  .\todo_app\test_app.py
 ```
+
+## DOCKER
+
+The application now resides as a container with to instances that can be run
+
+    - module5_todo:dev
+    - module5_todo:prod
+
+To build the containers you will need to build out the instances
+
+DEV instance
+```bash
+$ docker build --target development --tag todo-app:dev . --progress plain --no-cache
```

Comment:
I think the tag here doesn't quite match the one you assume elsewhere (and likewise for prod)
```suggestion
$ docker build --target development --tag module5_todo:dev . --progress plain --no-cache
```